lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingTransportServer.cs

```diff
@@ -71,19 +69,19 @@ public WritableTransportServer(int port, IObserver<TransportEvent<T>> remoteHand
         /// <param name="remoteHandler">The handler to invoke when receiving incoming
         /// remote messages</param>
         /// <param name="tcpPortProvider">Find port numbers if listenport is 0</param>
-        /// <param name="injector">The injector to pass arguments to incoming messages</param>
-        public WritableTransportServer(
+        /// <param name="streamingCodec">Streaming codec</param>
+        public StreamingTransportServer(
```

Comment:
You are right...There seems to be some complicated reduntant logic here..Let me try to clean this up...But it still needs LocalEndpoint to get IPAddress.